docs/user_guide/flowdecomposition.rst

```diff
@@ -0,0 +1,60 @@
+Running a flow decomposition
+============================
+
+.. testsetup:: *
+
+    import pathlib
+    import pandas as pd
+
+    import pypowsybl as pp
+    
+    pd.options.display.max_columns = None
+    pd.options.display.expand_frame_repr = False
+    import os
+    cwd = os.getcwd()
+    PROJECT_DIR = pathlib.Path(cwd).parent
+    DATA_DIR = PROJECT_DIR.joinpath('data')
+
+You can use the module :mod:`pypowsybl.flowdecomposition` in order to run load flows on networks.
+Please check out the examples below.
+
+For detailed documentation of involved classes and methods, please refer to the :mod:`API reference <pypowsybl.flowdecomposition>`.
+
+Start by importing the module:
+
+.. code-block:: python
+
+   import pypowsybl as pp
+
+Flow decomposition
+------------------
+
+To performa a flow decomposition, you need at least a network.
+The flow decomposition computer will return a dataframe containing the flow decomposition and the reference values.
```

Comment:
Better use present. The flow decomposition returns...

Reply:
the reference values -> the active power flows?

Reply:
done